astropy/units/quantity_helper/function_helpers.py

```diff
@@ -111,6 +111,12 @@
     # Array-API compatible versions (matrix axes always at end).
     SUBCLASS_SAFE_FUNCTIONS |= {np.linalg.diagonal, np.linalg.trace}
 
+    # works out of the box (tested)
+    SUBCLASS_SAFE_FUNCTIONS |= {np.linalg.cross}
+
+    # doesn't make much sense with quantites ?
+    SUBCLASS_SAFE_FUNCTIONS |= {np.linalg.svdvals}
```

Comment:
Maybe note that It works because it just calls `svd` internally, which transfers the unit (quantities are fine I think)

Reply:
Note that I actually didn't add a test yet, because the reference implementation lives in SciPy so it's a little bit more involved. But I'm working on it, and I'll add a comment at the same time, thanks for your suggestion !